README.md

```diff
@@ -197,9 +197,18 @@ OPTIONS:
         --solver-time-limit <solver-time-limit>
             The offset from the start of the batch to cap the solver's execution time [env: SOLVER_TIME_LIMIT=]
             [default: 210]
-        --solver-type <solver-type>
-            Which style of solver to use. Can be one of: 'NAIVE' for the naive solver; 'MIP' for mixed integer
-            programming solver; 'NLP' for non-linear programming solver [env: SOLVER_TYPE=]  [default: naive-solver]
+        --solver-config <solver-config>
+            Configuration for the solver. The possible solver types are:
+            'NaiveSolver' for the naive matching just two orders
+            'OpenSolver' for solutions on trade-pairs only
+            'StandardSolver' for solutions with ring-trades (mixed integer programming)
+
+            min_avg_fee_per_order allows to enforce economical viable solutions by
+            setting a minimum fee per order.
+
+            For example: {
+              "StandardSolver": { "min_avg_fee_per_order": 10000000000000000 }
```

Comment:
Do we know what value we will be using for this? I guess what you have here is 0.1 OWL

Reply:
Yeah, we were thinking about 0.01 OWL, but it's up to adjustments/discussions/investigations

Reply:
0.01 OWL seems very low considering the transaction costs of solution submission.